apps/meteor/client/views/omnichannel/departments/EditDepartment.js

```diff
@@ -59,6 +59,7 @@ function EditDepartment({ data, id, title, reload, allowedToForwardData }) {
 	const { department } = data || { department: {} };
 
 	const [[tags, tagsText], setTagsState] = useState(() => [department?.chatClosingTags ?? [], '']);
+	const [newTagsState, setNewTagsState] = useState(false);
```

Comment:
Wouldn't it be better if you named this const as `hasTagsChanged` or `hasNewTags`? I think having state in the name doesn't add much meaning to the variable.

Reply:
I don't think this is really necessary at this point... Probably this will be removed on rewrite, at the moment is just a workaround 😬